plugins/destination/clickhouse/typeconv/ch/types/data_type.go

```diff
@@ -0,0 +1,81 @@
+package types
+
+import (
+	"strconv"
+
+	"github.com/apache/arrow/go/v12/arrow"
+)
+
+func dataType(_type arrow.DataType) (string, error) {
+	switch _type.ID() {
```

Comment:
seems there is a mix between usage of `ID` and type casting. can we use just one? similar to https://github.com/cloudquery/cloudquery/blob/main/plugins/destination/postgresql/client/types_pg.go#L29 ?

Reply:
Good point, I'll look into it

Reply:
Done in fb53fc1